portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/frontenddataset/FrontendDataSetViews.testcase

```diff
@@ -92,4 +92,32 @@ definition {
 
 	}
 
+	@description = "LPS-179056 - Confirm that the user can create a data set view"
+	@priority = 5
+	test WithSpecialCharacters {
+		property custom.properties = "feature.flag.LPS-164563=true";
+
+		task ("Given access to Datasets admin page// App Menu > Control Panel > Datasets") {
+			LexiconEntry.gotoAdd();
+		}
+
+		task ("When The user goes to add Datasets // clicking on the New dataset plus button") {
+			PortletEntry.inputName(name = "~!@#$%^&*(){}[],.<>/?");
+		}
+
+		task ("And When you choose the Provider called channel ") {
+			FrontendDataSetAdmin.selectProvider(key_type = "Liferay Analytics Settings REST");
+		}
+
+		task ("And When clicking on the Save button") {
+			PortletEntry.save();
+		}
```

Comment:
You can cp these two (https://github.com/liferay-frontend/liferay-portal/commit/1ccf831a0e46f89e779baf69086b6c6645555ab3 https://github.com/liferay-frontend/liferay-portal/commit/d2b24b84fd4918cb361e963bc2203ff200c7c7f2 )commits and do it this way:
```suggestion
		FrontendDataSetAdmin.createDataSet(
				key_name = "~!@#$%^&*(){}[],.<>/?",
				key_type = "Channel");
```